.eslintrc.js

```diff
@@ -2,17 +2,868 @@
+        ['^\\.\\.(?!/?$)', '^\\.\\./?$'],
+
+        // Other relative imports. Put same-folder imports and `.` last.
+        ['^\\./(?=.*/)(?!/?$)', '^\\.(?!/?$)', '^\\./?$'],
+      ],
+    },
+  ],
+
+  'sentry/no-digits-in-tn': ['error'],
+
+  'sentry/no-dynamic-translations': ['error'],
+
+  // https://github.com/xojs/eslint-config-xo-typescript/blob/9791a067d6a119a21a4db72c02f1da95e25ffbb6/index.js#L95
+  '@typescript-eslint/no-restricted-types': [
+    'error',
+    {
+      types: {
+        // TODO(scttcper): Turn object on to make our types more strict
+        // object: {
+        //   message: 'The `object` type is hard to use. Use `Record<string, unknown>` instead. See: https://github.com/typescript-eslint/typescript-eslint/pull/848',
+        //   fixWith: 'Record<string, unknown>'
+        // },
+        Buffer: {
+          message:
+            'Use Uint8Array instead. See: https://sindresorhus.com/blog/goodbye-nodejs-buffer',
+          suggest: ['Uint8Array'],
+        },
+        '[]': "Don't use the empty array type `[]`. It only allows empty arrays. Use `SomeType[]` instead.",
+        '[[]]':
+          "Don't use `[[]]`. It only allows an array with a single element which is an empty array. Use `SomeType[][]` instead.",
+        '[[[]]]': "Don't use `[[[]]]`. Use `SomeType[][][]` instead.",
+      },
+    },
+  ],
+  // TODO(scttcper): Turn no-empty-object-type on to make our types more strict
+  // '@typescript-eslint/no-empty-object-type': 'error',
+  // TODO(scttcper): Turn no-function on to make our types more strict
+  // '@typescript-eslint/no-unsafe-function-type': 'error',
+  '@typescript-eslint/no-wrapper-object-types': 'error',
+
+  // Naming convention enforcements
+  '@typescript-eslint/naming-convention': [
+    'error',
+    {
+      selector: 'typeLike',
+      format: ['PascalCase'],
+      leadingUnderscore: 'allow',
+    },
+    {
+      selector: 'enumMember',
+      format: ['UPPER_CASE'],
+    },
+  ],
+
+  // Don't allow lookbehind expressions in regexp as they crash safari
+  // We've accidentally used lookbehinds a few times and caused problems.
+  'no-lookahead-lookbehind-regexp/no-lookahead-lookbehind-regexp': [
+    'error',
+    'no-lookbehind',
+    'no-negative-lookbehind',
+  ],
+};
+
+const strictRules = {
+  'no-console': ['error'],
+
+  // https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/no-is-mounted.md
+  'react/no-is-mounted': ['error'],
+
+  // https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/no-find-dom-node.md
+  // Recommended to use callback refs instead
+  'react/no-find-dom-node': ['error'],
+
+  // https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/no-string-refs.md
+  // This is now considered legacy, callback refs preferred
+  'react/no-string-refs': ['error'],
+
+  'jest/no-large-snapshots': ['error', {maxSize: 2000}],
+
+  'sentry/no-styled-shortcut': ['error'],
+};
+
+const extendsList = [
+  'sentry',
```

Comment:
i think its complaining about this

Reply:
d'oh, missed in [00fae01](https://github.com/getsentry/sentry/pull/80970/commits/00fae014631dc8a9eb67cf905bcb39076564b335)